src/components/Table/index.tsx

```diff
@@ -37,10 +37,16 @@ const Table = (props: Props) => {
         {tableContent.map((row, rowIndex) => (
           <tr
             className={onRowClick ? styles['clickable-row'] : styles['non-clickable-row']}
+            tabIndex={0}
```

Comment:
@viniciusaa ideally each row should have a different tabIndex value. You can make use of the `rowIndex` here and use it in place of the default 0

Reply:
Zero only means that I'm not defining its position, the browser will do it for me.